presto-docs/src/main/sphinx/release/release-0.214.rst

```diff
@@ -0,0 +1,63 @@
+=============
+Release 0.214
+=============
+
+General Changes
+---------------
+
+* Fix memory leak related to query tracking in coordinator that was introduced
+  in 0.213.
+* Send proper error response to client for certain types of errors during query
+  creation.
+* Improve error message when comparator provided to :func:`array_sort` function
+  doesn’t satisfy the required contract.
+* Improve query performance...??Lazily build hash table for ``MapBlock`` when
+  the hash table is used in ``SingleMapBlock``.
+* Improve planning more stable with regard to query predicates and joins.
```

Comment:
```suggestion
* Make planning more stable with regard to query predicates and joins.
```

Also, "more stable" is pretty vague for release notes. Maybe we just scrap this item?

Reply:
See [discussion](https://github.com/prestodb/presto/pull/11938#discussion_r234465250)